integrations/google-tag-manager/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@segment/analytics.js-integration-google-tag-manager",
   "description": "The Google Tag Manager analytics.js integration.",
-  "version": "2.5.0",
+  "version": "2.6.0-beta",
```

Comment:
Fun, is this for customer to validate before making it 'official'?

Reply:
Yes!